src/validation.cpp

```diff
@@ -1312,6 +1323,9 @@ bool CChainState::IsInitialBlockDownload() const
     if (m_chain.Tip()->GetBlockTime() < (GetTime() - nMaxTipAge))
         return true;
     LogPrintf("Leaving InitialBlockDownload (latching to false)\n");
+    f = fsbridge::fopen(latch_filename, "wb");
+    if (!f) LogPrintf("Failed to write %s \n", latch_filename.string().c_str());
+    else fclose(f);
```

Comment:
I think we now use bracket syntax for the conditional (after a famous Apple bug iirc)
```diff
-    if (!f) LogPrintf("Failed to write %s \n", latch_filename.c_str());
-    else fclose(f);
+    if (!f) {
+        LogPrintf("Failed to write %s \n", latch_filename.c_str());
+    } else {
+        fclose(f);
+    }
```

Reply:
Coding style guidelines doesn't agree, but I will expand it to new lines to make it easier to read.

(Indeed ironically I argued for using braces everywhere, but we don't).

https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c

Reply:
The developer notes state:
```
  - If an `if` only has a single-statement `then`-clause, it can appear
    on the same line as the `if`, without braces. In every other case,
    braces are required, and the `then` and `else` clauses must appear
    correctly indented on a new line.
```

Reply:
So if I read it correctly, without the `else` it would be fine, but with the `else` then braces are required.